src/games/hangman/util.ts

```diff
@@ -66,14 +66,30 @@ export function getMistakeCount(guesses: Guesses) {
   return count;
 }
 
-function getCorrectLettersCount(guesses: Guesses) {
+export function getCorrectLettersCount(guesses: Guesses) {
   let count = 0;
   for (const indexes of Object.values(guesses)) {
     count += indexes.length;
   }
   return count;
 }
 
+export function wasGuessCorrect(G: HangmanState, playerID: string) {
```

Comment:
Please rename method to isGuessCorrect... Generally we use "is" for functions that return boolean